### ISSUE TITLE
VcpkgPackage ordering issue

### ISSUE THREAD

**Issue by bengreenier** [bengreenier/vcpkg.nuget#22]
> from https://github.com/bengreenier/vcpkg.nuget/pull/21#issuecomment-437596098

@nefarius reported seeing an issue where importing the project first and defining packages afterward causes the packages to not be found - i don't believe that's expected behavior. As such, creating this issue to track investigation and a potential fix (could be a docs fix).

**Comment by nefarius** [comment]
IIRC it's not the package(s) not being detected but the triplets not getting through causing the build to fail. Here's how I currently use it with success (notice that I explicitly wanna build static):

```xml
  <PropertyGroup Label="VcpkgTriplets">
    <VcpkgTriplet Condition="'$(Platform)'=='Win32'">x86-windows-static</VcpkgTriplet>
    <VcpkgTriplet Condition="'$(Platform)'=='x64'">x64-windows-static</VcpkgTriplet>
  </PropertyGroup>
  <PropertyGroup>
    <VcpkgTaskTimeout>1200000</VcpkgTaskTimeout>
  </PropertyGroup>
  <Import Project="..\..\packages\Vcpkg.Nuget.1.5.0\build\Vcpkg.Nuget.props" Condition="Exists('..\..\packages\Vcpkg.Nuget.1.5.0\build\Vcpkg.Nuget.props')" />
  <ItemGroup Label="VcpkgPackages">
    <VcpkgPackage Include="poco" />
  </ItemGroup>
```

Will throw in some updates here what happens if I re-arrange.

**Comment by bengreenier** [comment]
ah - this seems to be expected. see https://github.com/Microsoft/vcpkg/blob/master/docs/users/integration.md#with-msbuild

TLDR, sounds like the designed path is to override triplets using globals:

```
<PropertyGroup Label="Globals">
  <!-- .... -->
  <VcpkgTriplet Condition="'$(Platform)'=='Win32'">x86-windows-static</VcpkgTriplet>
  <VcpkgTriplet Condition="'$(Platform)'=='x64'">x64-windows-static</VcpkgTriplet>
</PropertyGroup>
```

A quick local test shows that working for me
